### PR TITLE
PCHR-3436: Use Datepicker and text-angular from shoreditch

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR local copy of datepicker styles has been deleted from Job Contract and instead the styles from Shoreditch has been used to maintain consistency and reduce redundancy.

## Technical Details
1. `uk.co.compucorp.civicrm.tasksassignments/scss/civihr/partials/_datepicker.scss` has been removed.
2. Text angular styles has been removed from `uk.co.compucorp.civicrm.tasksassignments/scss/civihr/partials/_forms.scss`.
3. Uib-Datepicker and text-angular has been imported from shoreditch
```scss
@import '../../../org.civicrm.shoreditch/scss/bootstrap/overrides/variables';
@import '../../../org.civicrm.shoreditch/scss/bootstrap/mixins/prefix';
@import '../../../org.civicrm.shoreditch/scss/angular/overrides/uib-datepicker';
@import '../../../org.civicrm.shoreditch/scss/angular/overrides/text-angular';
```
T&A has a compass based build system and SCSSROOT is not integrated yet, so relative paths has been used above.

## Comments
After using the text-angular from shoreditch, the styles has changed but that is expected, it now matches the styles from Leave Tab.
**Before**
2018-03-21 at 10 34 am

**After**
2018-03-21 at 10 26 am

Backstop and Manual testing has been performed to ensure nothing is broken.

### CODE REVIEW

File: uk.co.compucorp.civicrm.tasksassignments/css/civitasks.css [hunk 1]
```diff
@@ -40,6 +40,7 @@
   /* add generic styling for the editor */
   /* add the styling for the awesomness of the resizer */
   /* copy the popover code from bootstrap so this will work even without it */
+  /* stylelint-disable selector-no-qualifying-type */
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/css/civitasks.css]: why is this necessary? The .css file should not be even considered by the linter.

File: uk.co.compucorp.civicrm.tasksassignments/scss/civihr/partials/_forms.scss [hunk 2]
```diff
@@ -32,6 +35,7 @@ input[type="checkbox"],
   }
 
   .has-feedback {
+    /* stylelint-disable selector-max-compound-selectors */
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civihr/partials/_forms.scss]: i think this should be `disable-line`

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks.scss [hunk 3]
```diff
@@ -1,142 +1,146 @@
...
+  @import 'civitasks/partials/spinner';
+  @import 'civitasks/partials/tables';
+  @import 'civitasks/partials/editable';
+  @import 'civitasks/partials/custom-select';
+  @import 'civitasks/partials/nav-tabs';
+
+  @import 'civitasks/modules/ie';
+  @import 'civitasks/modules/civihr-ui-select';
+
+  @import '../../../org.civicrm.shoreditch/scss/bootstrap/overrides/variables';
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks.scss]: use `SCSSROOT`

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss [hunk 4]
```diff
@@ -2,8 +2,8 @@
 
 
 // Include the original ui-select css + the civihr-ui-select dropdown theme
-@import "../../../../../org.civicrm.shoreditch/scss/bootstrap/vendor/ui-select/ui-select";
-@import "../../../../../civihr/org.civicrm.bootstrapcivihr/scss/components/civihr-ui-select/civihr-ui-select";
+@import '../../../../../org.civicrm.shoreditch/scss/angular/vendor/ui-select/ui-select';
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss]: use `SCSSROOT`

File: uk.co.compucorp.civicrm.tasksassignments/css/civitasks.css (hunk 1 above)
Comment (deb1990) [reply uk.co.compucorp.civicrm.tasksassignments/css/civitasks.css]: This is not added manually, after build process it is simply copied.

File: uk.co.compucorp.civicrm.tasksassignments/scss/civihr/partials/_forms.scss (hunk 2 above)
Comment (deb1990) [reply uk.co.compucorp.civicrm.tasksassignments/scss/civihr/partials/_forms.scss]: No inside this block, there are multiple instances of more than 3 compound selectors, so disabling all at once.

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks.scss (hunk 3 above)
Comment (deb1990) [reply uk.co.compucorp.civicrm.tasksassignments/scss/civitasks.scss]: T&A has a compass based build system and SCSSROOT is not integrated yet, so did not add that.

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss (hunk 4 above)
Comment (deb1990) [reply uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss]: T&A has a compass based build system and SCSSROOT is not integrated yet, so did not add that.
